### PR TITLE
HMIS-1209 Spike for terraform

### PR DESCRIPTION
**Pull request by junaidiqbalmoj**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/HMIS-1209

### Change description ###

Add multiple policy files for endpoints.

**Does this PR introduce a breaking change?** (check one with "x")
[ ] Yes
[x] No

### CODE REVIEW

File: .idea/.gitignore
Comment (joshblackmoor) [inline .idea/.gitignore:1]: .idea/ folder should be included in the gitignore

**Review by joshblackmoor — COMMENTED** [review]
Policy files will need updating but assuming that will happen in the later tickets

**Review by joshblackmoor — COMMENTED** [review]
You also need to do the infra approvals before merge. See message in pipeline regarding it

File: .idea/.gitignore
Comment (junaidiqbalmoj) [reply .idea/.gitignore:1]: added in gitignore.

**Review by junaidiqbalmoj — COMMENTED** [review]
sds infra changes have been approved and pipeline is green now.

**Review by joshblackmoor — COMMENTED** [review]
Code looks fine, can still see .idea folder though

File: resources/api-spec/hmi-api-health.json
Comment (ChrisS1512) [inline resources/api-spec/hmi-api-health.json:7]: Should this host be configurable / it replaces the dev with the relevant env it's running in?

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]

File: resources/api-spec/hmi-api-health.json
Comment (ChrisS1512) [inline resources/api-spec/hmi-api-health.json:22]: Can a description be added for this

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]

File: resources/policy-files/CaTH/publication.xml
Comment (ChrisS1512) [inline resources/policy-files/CaTH/publication.xml:4]: Similar here (and the api-policy.xml file)

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]

**Review by joshblackmoor — APPROVED** [review]

**Pull request merged**
